### PR TITLE
Added favicon.

### PR DESCRIPTION
**Pull request by biocom**

### CODE REVIEW

**Comment by biocom** [comment]
I tried setting the .SVG file in webpack, but it throws an error (with the underlaying sharp library). If you can fix it to use the .SVG great, otherwise the .PNG is currently in place and working.

@anxolin Also please review the app name/description.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr585--dexreact.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr585--dexreact.review.gnosisdev.com

**Comment by biocom** [comment]
@Velenir Just did that now and that seemed to work. Thank you.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr585--dexreact.review.gnosisdev.com

File: webpack.config.babel.js [hunk 1]
```diff
@@ -91,7 +92,7 @@ module.exports = ({ stats = false } = {}) => ({
   plugins: [
     new HtmlWebPackPlugin({
       template: './src/html/index.html',
-      title: 'dFusion',
+      title: 'Gnosis Protocol',
```
Comment (alfetopito) [inline webpack.config.babel.js]: My understanding is that that for now we'll be using `dFusion` name and switch to `Gnosis Protocol` when we publicly launch.

@c3rnst

Comment (anxolin) [reply webpack.config.babel.js]: I let @c3rnst answer that :)

**Comment by biocom** [comment]
Updated text to dFusion and added a 'dF' logo icon.

**Comment by anxolin** [comment]
Cool! fill free to Squash and merge to `develop` :)

**Comment by biocom** [comment]
@anxolin Done, thanks :)

**Pull request merged**